### DISCUSSION TITLE
[Feature] immich app should show local photos, even if immich server is not available

### DISCUSSION THREAD

**Discussion by bavarianbidi** [immich-app/immich#13906] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

My immich server instance was not available due to an issue (which was unrelated to immich itself). Unfortunately i was on vacation and with that AFK for more than a week to fix the server configuration in my home lab.

That circumstance brought the downside, that on my mobile phone (Android), i was not able to use immich as "default image viewer" to see the recent pictures i've taken with the camera.
I had to fix the server first to be able to see the images in immich mobile app.

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2266/#4466

**Comment by bavarianbidi** [comment]
xref: https://github.com/immich-app/immich/discussions/2266#discussioncomment-11141162
